README.md

```diff
@@ -2,111 +2,78 @@
 
 [![Discord](https://img.shields.io/discord/559159668438728723.svg?logo=discord)](https://discord.gg/2TsNFKt)
 
-**stash-box is Stash App's own OpenSource video indexing and Perceptual Hashing MetaData API server for porn.**
+Stash-box is an open-source video indexing and metadata API server for porn developed by Stash App. The purpose of stash-box is to provide a community-driven database of porn metadata, similar to what MusicBrainz does for music. The submission and editing of metadata should follow the same principles as MusicBrainz. [Learn more here](https://musicbrainz.org/doc/Editing_FAQ). Installing Stash-box will create an empty database for you to populate.
```

Comment:
Removed the phrase `perceptual hashing`. I would say that it's worth mentioning as it's one of the core aspects.

Reply:
The introduction aimed for something a bit more human digestible of you want to propose an additional PR that re-includes this I can merge it but I think a high level abstraction is a bit more digestible.  Doesn't mean I want it gone but I think on this PR I'm going to leave it unless theres a suggestion.